core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -26,7 +26,10 @@ import scala.util.control.{ControlThrowable, NonFatal}
 import com.codahale.metrics.{Gauge, MetricRegistry}
 
 import org.apache.spark.internal.Logging
-import org.apache.spark.internal.config.{DYN_ALLOCATION_MAX_EXECUTORS, DYN_ALLOCATION_MIN_EXECUTORS}
+import org.apache.spark.internal.config.{
```

Comment:
I would just make this import org.apache.spark.internal.config._

Reply:
Done